### PR TITLE
use max_cromwell_retries config field to set cromwell options

### PR DESCRIPTION
**Pull request by charlesdyan**
### Purpose

[JIRA](https://broadinstitute.atlassian.net/browse/GH-282)

### Changes

use max_cromwell_retries config field to set cromwell options for workflows
[parallel workflow changes in skylab](https://github.com/HumanCellAtlas/skylab/pull/226)
[parallel workflow changes in pipeline-tools](https://github.com/HumanCellAtlas/pipeline-tools/pull/151)

### Review Instructions

- No instructions.

### CODE REVIEW

File: lira/test/test_lira_utils.py [hunk 1]
```diff
@@ -449,6 +449,13 @@ def test_compose_caas_options(self):
             fake_caas_key_json['client_email'],
         )
 
+    def test_compose_config_options(self):
```
Comment (samanehsan) [inline lira/test/test_lira_utils.py]: Can you add a test case where a value for `maxRetries` is specified in the options file?

File: lira/lira_utils.py [hunk 2]
```diff
@@ -219,6 +218,18 @@ def compose_caas_options(cromwell_options_file, lira_config):
...
 
 
+def compose_config_options(cromwell_options_file, lira_config):
+    if isinstance(cromwell_options_file, bytes):
+        cromwell_options_file = cromwell_options_file.decode()
+    options_json = json.loads(cromwell_options_file)
+
+    # defer to value already in options file if it exists
+    if 'maxRetries' not in options_json.keys():
+        options_json['maxRetries'] = lira_config.max_cromwell_retries
```
Comment (samanehsan) [inline lira/lira_utils.py]: I think the options file would actually look like this, but I'm not entirely sure from looking at the [cromwell docs](https://cromwell.readthedocs.io/en/latest/wf_options/Overview/#workflow-options-overview):
```
{
    "default_runtime_attributes": {
        "maxRetries": 3
    }
}
```

**Comment by samanehsan** [comment]
Left a few comments - let me know if you have any questions! :octocat: ☀️

File: lira/lira_utils.py (hunk 2 above)
Comment (samanehsan) [reply lira/lira_utils.py]: Update: I tried running a workflow in cromwell with these workflow options and it worked!

**Review by rexwangcc — APPROVED** [review]
lgtm

**Comment by samanehsan** [comment]
The integration test succeeded but for some reason the status hasn't changed on the PR: https://gotc-jenkins.dsp-techops.broadinstitute.org/job/integration-tests/2539/console

**Pull request merged**